snet-ui/Blockchain/WalletModal.tsx

```diff
@@ -100,42 +91,41 @@ export default function WalletModal({ open, setOpen }: Props) {
   useEagerConnect();
 
   const handleConnect = async (connector: AbstractConnector | undefined) => {
-    let name = "";
+    let name = '';
     Object.keys(SUPPORTED_WALLETS).map((key) => {
       if (connector === SUPPORTED_WALLETS[key].connector) {
         return (name = SUPPORTED_WALLETS[key].name);
       }
       return true;
     });
 
-    console.log("connecting to the wallet", name);
+    console.log('connecting to the wallet', name);
 
     // if the connector is walletconnect and the user has already tried to connect, manually reset the connector
-    if (
-      connector instanceof WalletConnectConnector &&
-      connector.walletConnectProvider?.wc?.uri
-    ) {
+    if (connector instanceof WalletConnectConnector && connector.walletConnectProvider?.wc?.uri) {
       connector.walletConnectProvider = undefined;
     }
 
-    connector &&
+    if (connector) {
       activate(connector, undefined, true)
```

Comment:
@rajeevbbqq this is the main fix I made. Please check this

Reply:
@raamb This won't make any changes